tests/php/Extension/FluentExtensionTest/TestGeneratedColumns.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace TractorCow\Fluent\Tests\Extension\FluentExtensionTest;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DataObject;
+use TractorCow\Fluent\Extension\FluentExtension;
+
+/**
+ * @mixin FluentExtension
+ */
+class TestGeneratedColumns extends DataObject implements TestOnly
+{
+    private static string $table_name = 'FluentExtensionTest_TestGeneratedColumns';
+
+    private static array $db = [
+        'BaseField' => 'Varchar(255)',
+        'GeneratedField1' => 'Generated("Varchar(255)", "CONCAT(\\"BaseField\\", \'_virtual\')", "VIRTUAL")',
+        'GeneratedField2' => 'Generated("Varchar(255)", "CONCAT(\\"BaseField\\", \'_stored\')", "STORED")',
```

Comment:
Is "Generated" some new silverstripe column type?

Reply:
Yes - the details are in the linked issue and the linked framework PR